EKF/ekf_helper.cpp

```diff
@@ -223,28 +223,44 @@ void Ekf::resetHorizontalPositionTo(const Vector2f &new_horz_pos)
 	_state_reset_status.posNE_counter++;
 }
 
+void Ekf::resetVerticalPositionTo(const float &new_vert_pos)
+{
+	printf("ecl: new vert pos: %f", (double)new_vert_pos);
```

Comment:
Change to ECL_INFO or remove.

Reply:
Thanks for spotting that printf() degbugging, it' wasn't meant to stay in. :}

Reply:
Fixed and pushed up.